packages/list/ListEditing.js

```diff
@@ -78,6 +60,7 @@ ListEditing.Prototype = function() {
           path: path,
           startOffset: 0
         });
+				args.selection = selection;
```

Comment:
It's actually meant to be 4 tabs with each tab as 2 spaces. But Github is interpreting each tab as 4 spaces probably because of the mixed use of tabs and spaces. I'll just convert the tabs to spaces.